client/actions/auth/types.js

```diff
@@ -2,3 +2,4 @@ export const APP_LOAD = 'APP_LOAD'
 export const LOGIN = 'LOGIN'
 export const LOGOUT = 'LOGOUT'
 export const FORGOT_PASSWORD = 'FORGOT_PASSWORD'
+export const LOGIN_FAILED = 'LOGIN_FAILED'
```

Comment:
let's either go with `FAILED_LOGIN` or change the project action types to prefix with `PROJECTS` to stick with consistent naming conventions